packages/libx264/build.sh

```diff
@@ -2,33 +2,30 @@ TERMUX_PKG_HOMEPAGE=https://www.videolan.org/developers/x264.html
 TERMUX_PKG_DESCRIPTION="Library for encoding video streams into the H.264/MPEG-4 AVC format"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-_COMMIT=b093bbe7d9bc642c8f24067cbdcc73bb43562eab
-TERMUX_PKG_VERSION=1:0.164.3101 # X264_BUILD from x264.h; commit count
+_COMMIT=4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1
+# X264_BUILD from x264.h; commit count using "git rev-list --count HEAD" on x264 git repo
```

Comment:
Would it be possible to get that version style automatically like in luajit https://github.com/termux/termux-packages/blob/2f56828d5ca2077d6601acfb9c265f55d0447d4e/packages/libluajit/build.sh#L16 ?

Reply:
Code in luajit gets commit from version, it is not applicable in this case.